core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/CompactEntityComponent.scala

```diff
@@ -33,7 +33,10 @@ trait CompactEntityComponent extends LazyLogging {
   object compactEntityQuery extends CompactEntityQuery(this)
 }
 
-class CompactEntityQuery(driverComponent: DriverComponent) extends RawSqlQuery with CompactEntitySerialization {
+class CompactEntityQuery(driverComponent: DriverComponent)
+    extends CompactEntityMigration
+    with RawSqlQuery
+    with CompactEntitySerialization {
```

Comment:
I moved the migration-related SQL to the new `CompactEntityMigration` trait to try to stay organized.